Confus/Confus/Game.cpp

```diff
@@ -246,8 +246,7 @@ namespace Confus
     void Game::fixedUpdate()
     {
 		m_MazeGenerator.fixedUpdate();
-		m_PhysicsWorld.stepSimulation(static_cast<float>(FixedUpdateInterval));
-		m_PlayerHandler.fixedUpdate();
+		//m_PhysicsWorld.stepSimulation(static_cast<float>(FixedUpdateInterval));
```

Comment:
Can you remove the line? Not so bothered with it being commented out, but will conflict with my change :) (also, why is the fixed update on the PlayerHandler removed?)